tests/output/test_bazefetcher.py

```diff
@@ -6,17 +6,27 @@
 import os
 import pandas as pd
 import pytest
+from camille.util import utcdate
+import sys
+
 
 eps = timedelta(microseconds=1)
 
+
+def assert_series_equal_wrap(_arg1, _arg2):
+    if sys.version_info[1] < 6:
+        pd.testing.assert_series_equal(_arg1, _arg2)
+    else: 
+        pd.testing.assert_series_equal(_arg1, _arg2, check_freq=False)
+
```

Comment:
I saw this function in multiple places. It would be great if we could avoid code duplication. Can we move it to some separate module and import from it?

Wouldn't it be better to check if function has the parameter 'check_freq' instead of just general checking of python version?